features/step_definitions/clear_data_on_signout_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
+  visit('/en/tools/money-manager/')
+end
+
+Then(/^I complete the series of questions$/) do
+  find_field(id: 'questionnaire_received_first_payment_true').click
```

Comment:
Hey @picandocodigo how do you feel about using a [_page object model DSL for Capybara_](https://github.com/natritmeyer/site_prism) for cleaning up some of this code, please?